### DISCUSSION TITLE
Exiting osu!lazer to update cuts off announcer's "See you next time" to "See you n..."

### DISCUSSION THREAD

**Discussion by Skubers** [ppy/osu#18399] [Q&A] [answered]
### Type

Game behaviour

### Bug description

Same as title. I've updated osu!lazer to newest version today and this happened. I unfortunately don't have a video since I don't have an older version anymore.

### Version

2022.319.0-lazer -> 2022.523.0-lazer

### Logs

Can't find Logs folder for lazer

**Comment by peppy** [comment]
I can't reproduce this.

Are you potentially hitting alt-f4 twice or something? How are you triggering the exit process?

Also please try and find logs using this help article: https://github.com/ppy/osu/discussions/15188

**Reply by Gabixel** [reply]
I think this is related to https://github.com/ppy/osu/issues/10630 since this "exit timer" seems to be triggered even after agreeing to restart the game, so it forcibly skips the exit animation at any time.

**Reply by Gabixel** [reply]
It can be reproduced easily if you click on the confirm button just before this "exit timer" trigger.
Here's an example of the issue:

https://user-images.githubusercontent.com/43073074/170024807-402edf42-8df9-4a21-ba81-9da757ad2468.mp4

**Reply by Skubers** [reply]
exactly this behaviour

**Comment by Skubers** [comment]
![image](https://user-images.githubusercontent.com/82095491/170021134-3f74720f-20c0-4509-b164-584cfd07701a.png)
No Logs folder here

Also, I'm sure I wasn't hitting the alt-f4 button. I just clicked the notification that an update was ready, then the exit prompt appeared and I clicked the button to exit

**Reply by integerrr** [reply]
Have you perhaps changed install directory? If so go to in game setting and press "open osu! Folder" instead for your logs folder

**Reply by Skubers** [reply]
oh right, that's the thing. sorry
[logs.zip](https://github.com/ppy/osu/files/8762132/logs.zip)

**Comment by Wufler** [comment]
Happens to me when i click "restart to update" in the notification center and when the popup to make sure if you wanna exit out of osu! I click it myself and cuts off the "see you next time"

**Comment by peppy** [comment, marked as answer]
Will be resolved by https://github.com/ppy/osu/pull/18760.
